specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2021-04-01-preview/definitions.json

```diff
@@ -6493,6 +6493,89 @@
         }
       }
     },
+    "PrivateEndpointConnectionRequest": {
+      "description": "A request to approve or reject a private endpoint connection",
+      "type": "object",
+      "properties": {
+        "Id": {
```

Comment:
> "Id" [](http://example.com/codeflow?start=7&length=5)

Why Id is needed in the PUT request body? It's redundant since the request URL is the id.

Reply:
hi, for this one, I just followed nrp's documentation spec example

Reply:
![image](https://user-images.githubusercontent.com/29927264/127922518-36bfe82b-ca38-4f02-99d7-27098e4c33b1.png)

Reply:
I agree with Jin: there is something wrong here. I can't tell anything from that image. The PUT property bag has nothing in it but provisioning state and the body has an additional redundant id. It should have properties for creating the new private endpoint connection, something like PrivateEndpointConnectionWrapperProperties.  BTW, if the Id property remains here, it should be lowerCamelCase ("id").

Reply:
sure, will update it to lower case, the image I added here is nrp's official document, they suggested adding this id in the put operation's body...